packages/forklift-console-plugin/src/modules/Providers/utils/validators/secret-fields/openstackSecretFieldValidator.ts

```diff
@@ -42,9 +42,6 @@ export const openstackSecretFieldValidator = (id: string, value: string) => {
     case 'projectID':
       validationState = validateProjectID(trimmedValue) ? 'success' : 'error';
       break;
-    case 'userDomainName':
-      validationState = validateUserDomainName(trimmedValue) ? 'success' : 'error';
-      break;
```

Comment:
Shouldn't the `domainName` be validated? I see you dropped `userDomainName` validation but you didn't add the equivalent for `domainName`...

Reply:
it is:
https://github.com/kubev2v/forklift-console-plugin/blob/main/packages/forklift-console-plugin/src/modules/Providers/utils/validators/secret-fields/openstackSecretFieldValidator.ts#L33